renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -753,13 +753,23 @@ WrappedOpenGL::~WrappedOpenGL()
   GetResourceManager()->ClearReferencedResources();
 
   GetResourceManager()->ReleaseCurrentResource(m_DeviceResourceID);
-  GetResourceManager()->ReleaseCurrentResource(m_ContextResourceID);
+
+  for(auto it = m_ContextData.begin(); it != m_ContextData.end(); ++it)
+  {
+    if(it->second.m_ContextDataRecord)
+    {
+      RDCASSERT(it->second.m_ContextDataRecord->GetRefCount() == 1);
+      it->second.m_ContextDataRecord->Delete(GetResourceManager());
+      GetResourceManager()->ReleaseCurrentResource(it->second.m_ContextDataResourceID);
```

Comment:
Thinking about it further - we'll also need to delete this record when the context gets destroyed, otherwise if contexts are created and destroyed records will hang around and there's a chance at an overlap if the same context pointer is re-used. Just adding this code in `WrappedOpenGL::DeleteContext` should do the trick I think.